lib/topological_inventory/ansible_tower/receptor/api.rb

```diff
@@ -23,10 +27,10 @@ def class_from_type(type)
         send("#{type}_class") if respond_to?("#{type}_class")
       end
 
-      # JobTemplate v1 is for Tower version < 3 which we don't support
+      # JobTemplate v2 is for Tower version < 3 which we don't support
       # https://bugzilla.redhat.com/show_bug.cgi?id=1369842
       def job_template_class
-        @job_template_class ||= AnsibleTowerClient::JobTemplateV2
```

Comment:
What prompted this change?

Reply:
@syncrou I misunderstood the condition before, V2 is for older towers, V1 for >=3
https://github.com/ansible/ansible_tower_client_ruby/blob/master/lib/ansible_tower_client/api.rb#L197